portfolio/src/main/webapp/script.js

```diff
@@ -81,7 +94,7 @@ window.addEventListener("scroll", e => {
     if (fromTop > 300) {
         returnLink.classList.remove("top")
     } else {
-        returnLink.classList.add("top")
+        if (returnLink) returnLink.classList.add("top")
```

Comment:
Google jave style guide recommends using brackets even when optional. Ihttps://google.github.io/styleguide/javaguide.html#s4.1-braces

Reply:
ok thanks, I'll fix this!